packages/gatsby-admin/src/pages/index.tsx

```diff
@@ -1,5 +1,72 @@
 import React from "react"
-import { useQuery } from "urql"
+import { useQuery, useMutation } from "urql"
+
+const InstallInput = () => {
+  const [value, setValue] = React.useState("")
+
+  const [_, installGatbyPlugin] = useMutation(`
+    mutation installGatsbyPlugin($name: String!) {
+      createNpmPackage(npmPackage: {
+        name: $name,
+        dependencyType: "production"
+      }) {
+        id
+        name
+      }
+      createGatsbyPlugin(gatsbyPlugin: {
+        name: $name
+      }) {
+        id
+        name
+      }
+    }
+  `)
+
+  return (
+    <form
+      onSubmit={evt => {
+        evt.preventDefault()
+        installGatbyPlugin({
+          name: value,
+        })
+      }}
+    >
+      <label>
+        Install:
+        <input
+          value={value}
+          onChange={evt => setValue(evt.target.value)}
+          type="text"
+          placeholder="gatsby-plugin-cool"
+        />
+      </label>
+    </form>
+  )
+}
+
+const DestroyButton = ({ name }) => {
+  const [_, deleteGatsbyPlugin] = useMutation(`
+    mutation destroyGatsbyPlugin($name: String!) {
+      destroyNpmPackage(npmPackage: {
+        name: $name,
+        id: $name,
+        dependencyType: "production"
+      }) {
+        id
+        name
+      }
+      destroyGatsbyPlugin(gatsbyPlugin: {
+        name: $name,
+        id: $name
+      }) {
+        id
+        name
+      }
+    }
+  `)
+
+  return <button onClick={() => deleteGatsbyPlugin({ name })}>X</button>
+}
 
 export default (): React.ReactElement => {
```

Comment:
If you give this a name and export that as default afterwards it should show up properly in devtools. As above, typing it as React.FC will give correct return and props values